static/styles/portico-signin.css

```diff
@@ -586,19 +588,22 @@ button.login-github-button {
     color: #fff;
 }
 
+button.login-wikimedia-button,
 button.login-google-button,
 button.login-github-button {
     background-color: #fff;
     box-shadow: 0px 1px 3px hsla(0, 0%, 0%, 0.3), 0px 0px 5px hsla(0, 0%, 0%, 0.1);
     color: hsl(0, 0%, 34%);
 }
 
+button.login-wikimedia-button:hover,
 button.login-google-button:hover,
 button.login-github-button:hover {
     background-color: hsl(0, 0%, 98%);
     box-shadow: 1px 2px 5px hsla(0, 0%, 0%, 0.2);
 }
 
+button.login-wikimedia-button:active,
 button.login-google-button:active,
 button.login-github-button:active {
```

Comment:
For these 3, maybe it's worth putting a CSS class that we have on all these social buttons that can have these properties, rather than needing to add each one here?

(I'd just do the refactor in a commit at the bottom of the branch).

Reply:
You want me to do this as well?

Reply:
Yeah, that'd be great.  I'm hoping we can after these refactors have a relatively simple series for MediaWiki that we can point to for "how to add a new auth integration", which is why I'm thinking we should do these refactors at the bottom of the branch.

Reply:
Yep, I agree.